sklearn/datasets/california_housing.py

```diff
@@ -89,17 +87,20 @@ def fetch_california_housing(data_home=None, download_if_missing=True):
     if not exists(data_home):
         makedirs(data_home)
 
-    filepath = _pkl_filepath(data_home, TARGET_FILENAME)
+    filepath = _pkl_filepath(data_home, 'cal_housing.pkz')
     if not exists(filepath):
         if not download_if_missing:
             raise IOError("Data not found and `download_if_missing` is False")
 
-        print('downloading Cal. housing from %s to %s' % (DATA_URL, data_home))
-        archive_fileobj = BytesIO(urlopen(DATA_URL).read())
+        print('downloading Cal. housing from {} to {}'.format(
```

Comment:
Later on you are using the logger for this type of message. Do you want to make it uniform along the PR?